packages/nouns-api/src/graphql/schemas/schema.graphql

```diff
@@ -112,6 +112,8 @@ type Idea {
   createdAt: String!
   ideaStats: IdeaStats
   tags: [IdeaTags!]
+  archived: Boolean!
+  consensus: Float
```

Comment:
Havn't fully dug in, but do we not want to considered archived and consensus as "virtual tags" the same way we are treating "HOT" on my branch?

Reply:
We might want to add a virtual tag for it later if we add filters for different consensus ranges e.g. `> 10% consensus` but for now i think we just make the property available here and build out the tag manually on the client.